server/routes/stock-news.js

```diff
@@ -1,19 +1,52 @@
 var express = require('express');
 var router = express.Router();
 
-let newsSources = [
-    "Bloomberg",
-    "Financial Post",
-    "Financial Times",
-    "The Economist",
-    "The Global and Mail",
-    "The New York Times",
-    "The Wall Street Journal",
-    "Time"
+let newsSelections = [
+    {
+        id: "0",
+        name: "Bloomberg",
+        selected: true
+    },
+    {
+        id: "1",
+        name: "Financial Post",
+        selected: true
+    },
+    {
+        id: "2",
+        name: "Financial Times",
+        selected: true
+    },
+    {
+        id: "3",
+        name: "The Economist",
+        selected: true
+    },
+    {
+        id: "4",
+        name: "The Global and Mail",
+        selected: true
+    },
+    {
+        id: "5",
+        name: "The New York Times",
+        selected: true
+    },
+    {
+        id: "6",
+        name: "The Wall Street Journal",
+        selected: true
+    },
+    {
+        id: "7",
+        name: "Time",
+        selected: true
+    }
 ]
 
-const newsInfo = [
+let newsMasterlist = [
```

Comment:
Why are we using let instead of const

Reply:
Unless it's a changing variable. If it's static you can use const.

Reply:
will fix as discussed in meeting, thanks